website/docs/reference/references-overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "About References"
+id: "references-overview"
+sidebar_label: "About References"
+description: "Connect dbt to any data platform in dbt Cloud or dbt Core, using a dedicated adapter plugin"
+hide_table_of_contents: true
+---
+
+The References section contains reference materials for developing in dbt Cloud and dbt Core. 
+
+Learn how you can add more configurations to your dbt project or adapter, use properties for extra ability, refer to dbt commands, use powerful Jinja functions to streamline your dbt project, and understand how to use dbt artifacts. The <br />
+
+<div className="grid--3-col">
+
+<Card
+    title="Project configurations"
+    body="Customize and configure your dbt project to optimize performance."
+    link="/reference/dbt_project.yml"
+    icon="computer"/>
+
+<Card
+    title="Platform-specific configurations"
+    body="Learn how to do data platform-specific things in dbt Cloud and dbt Core to optimize performance."
+    link="/reference/resource-configs/postgres-configs"
+    icon="computer"/>
+
+<Card
+    title="Resource configurations and properties"
+    body="Your project's resources, such as models, snapshots, seeds, and tests, can have properties and configurations that provide extra abilities."
```

Comment:
It might benefit the reader more if we lead with the meat of what's in this section.  Maybe something like: 
```suggestion
    body="Properties and configurations that provide extra abilities to your projects resources."
```